change/@ni-nimble-components-e81831d9-2db3-450d-b3a8-776ab5156e86.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Let's update the matrix stories for affected components (select, combobox, listbox) to include an option that's truncated.